File: 15_InterrompendoWhile/teste/Teste.py
# cont = 1
# while cont <= 10:
#     print(cont, " -> ", end="")
#     cont += 1
# print("Acabou")


# numero = 0
# while numero != 999:
#     numero = int(input("Digite um numero: ")) 



# numero = contador = 0
# while contador < 10:
#     num = int(input("Digite um numero: "))
#     contador += 1

numero = soma = 0
while True:
    numero= int(input("Digite um numero"))
    if numero == 999:
       break
    soma += numero
print(f"[A soma  vale {soma}]")
        

File: teste.java
import java.util.Scanner;
import java.util.ArrayList;
import java.util.List;

public class teste{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        List <Number> numeros = new ArrayList<>();

        System.out.print("Digite o primeiro número: ");

        int num1 = input.nextInt();
        numeros.add(num1);

        System.out.print("Digite o segundo número: ");

        int num2 = input.nextInt();
        numeros.add(num2);
        
        int soma = num1 + num2;

        int media = (num1 + num2) / numeros.size();

        System.out.println("A soma é: " + soma);
        System.out.println("A média é: " + media);
        // System.out.println("O primeiro numero da lista é: " numeros.get(0));
        System.out.println("O tamanho da lista é: " + numeros.size());
    }
}
// public class Main {
//     public static void main(String [] args){
//         System.out.println("Olá mundo!");
//     }
// }


// public class somar{
//     public static void main(String[] args){
//         System.out.println("Olá mundo!");

//         int num1 = 5;
//         int num2 = 6;
//         int soma = num1 + num2;

//         System.out.println("A soma é: " + soma);
//     }
    

// }





